src/cli.rs

```diff
@@ -631,7 +593,7 @@ impl Cli {
             .collect();
 
         app.args(negations)
-            .after_help("Each option can be reset with a --no-OPTION argument.")
+            .after_help(format!("Each option can be reset with a --no-OPTION argument.\n\nRun \"{} help\" for more complete documentation.", env!("CARGO_PKG_NAME")))
```

Comment:
We used to show the `xh help` suggestion only if `--help` were used. Is that possible?